assets/js/modules/tagmanager/components/common/AMPContainerSelect.test.js

```diff
@@ -135,6 +147,8 @@ describe( 'AMPContainerSelect', () => {
 		const accountID = account.accountId; // eslint-disable-line sitekit/camelcase-acronyms
 		registry.dispatch( STORE_NAME ).receiveGetAccounts( [ account ] );
 		registry.dispatch( STORE_NAME ).setAccountID( accountID );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getContainers', [ accountID ] );
```

Comment:
Why is this marked as finished here? I'd think this should explicitly _not_ be finished here, to trigger the loading state in the UI.

Reply:
Good catch! Thanks! Updated to fix this issue.